package.json

```diff
@@ -53,8 +54,10 @@
     "build-css": "node-sass src/ -o src/",
     "watch-css": "npm run build-css && node-sass src/ -o src/ --watch --recursive",
     "start-js": "react-scripts start",
-    "start": "npm-run-all -p build watch-css start-js",
-    "build": "npm run build-css && npm run copy-monaco && react-scripts build",
+    "start": "env-cmd --fallback ./.env.development npm-run-all -p build watch-css start-js",
+    "start:local": "env-cmd --fallback ./.env.local npm-run-all -p build watch-css start-js",
+    "build": "env-cmd --fallback ./.env.production npm run build-css && npm run copy-monaco && react-scripts build",
+    "build:dev": "env-cmd --fallback ./.env.development npm run build-css && npm run copy-monaco && react-scripts build",
```

Comment:
there is likely a bunch I am missing here. where/how/when is `build:dev` used?

I suspect that most of my questions will be resolved by some examples as mentioned above.